ecs-init/config/common.go

```diff
@@ -44,17 +47,25 @@ const (
 	DefaultRegionName = "us-east-1"
```

Comment:
Is this unused now?

Reply:
It's still used as a default if for some unknown reason the region isn't mapped to any of the partitions.